### PR TITLE
OOP school library: add basic UI 

### PR DESCRIPTION
**Pull request by LoviMadaan**
### In this branch, I have added the following;
 

- [ ] Create an app.rb file that will serve as my console app entry point. It has methods that do the following:

- List all books.
- List all people.
- Create a person (teacher or student, not a plain Person).
- Create a book.
- Create a rental.
- List all rentals for a given person id.

 

- [ ] In the main.rb define the entry point, this will be a method called main that is invoked at the end of your file. This method should do the following:

- Present the user with a list of options to perform.
- Lets users choose an option.
- If needed, ask for parameters for the option.
- Have a way to quit the app.

### CODE REVIEW

File: student.rb [hunk 1]
```diff
@@ -1,5 +1,4 @@
-require_relative 'person'
-
+require_relative 'Person'
```
Comment (Ismailco) [inline student.rb]: - [x] Nice work creating the app however the `Person` should be `person` here as the name of the file as for now the app produces an error 👇🏼, but after changing the name the app works as expected. kindly consider making the change so your app works properly

Screenshot_20230603_150830

**Review by Ismailco — COMMENTED** [review]
### Hi @LoviMadaan, 

Good job so far 👍🏼 
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights

✔️ Create the App & main
✔️ No linter errors
✔️ Used Gitflow
✔️ Work documented professionally

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: student.rb (hunk 1 above)
Comment (LoviMadaan) [reply student.rb]: Thank you so much.. I have implemented the change.

**Review by KDlamini — APPROVED** [review]
### Hi @LoviMadaan 🙋‍♀️,  

Excellent work! 💯 
You have done a great job adding the Basic UI for the OOP school library. 🎊 🥳 
Your feature is complete! There is nothing else to say other than... it's time to merge it. :shipit: 
Congratulations!!!

## Highlights 🌞 

✔️  Good PR title and description
✔️  The `app.rb` and `main.rb` files are set up correctly
✔️  User can create students, books, and add rentals successfully
✔️  Well documented README

# Status: Approved ✔️ 

oscars-standing-ovation
 
Cheers 🥂  and Happy coding!!! 👯 
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @KDlamini  in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**